contracts/TokenConverter/RiskFundConverter.sol

```diff
@@ -268,7 +268,7 @@ contract RiskFundConverter is AbstractTokenConverter {
             for (uint256 i; i < poolsLength; ++i) {
                 uint256 poolShare = (poolsAssetsReserves[pools[i]][tokenAddress] * EXP_SCALE) / assetReserve;
                 if (poolShare == 0) continue;
```

Comment:
Would it be enough if we check in the `if` statement the value of `poolsAssetsReserves[pools[i]][tokenAddress]`? This way we could save the calculation to set `poolShare`

Reply:
https://github.com/VenusProtocol/protocol-reserve/pull/29/commits/7f289d05c6993f8c24dea0fbeba536df33d428fd